.github/workflows/docker.yaml

```diff
@@ -20,10 +20,6 @@ concurrency:
 
 on:
   workflow_call:
-    secrets:
```

Comment:
this may break guillaumes' repositories docker builds, i will fix if ever needed

Reply:
> (on the other side, would be very happy to remove submodules...)

You would have all my 4 thumbs up on this